src/services/fileStorage/strategies/interface.js

```diff
@@ -9,6 +9,10 @@ class AbstractFileStorageStrategy {
 		throw new TypeError('connect method has to be implemented.');
 	}
 
+	getBucket() {
+		throw new TypeError('get bucket method has to be implemented');
```

Comment:
would be helpful to have in the error the exact name of the method (getBucket instead of get bucket)

Reply:
```suggestion
		throw new TypeError('getBucket method has to be implemented');
```